core/application/react-query-adapter/project/client/use-submit-issue.ts

```
@@ -8,6 +8,8 @@ import { bootstrap } from "@/core/bootstrap";
 import { ProjectStoragePort } from "@/core/domain/project/outputs/project-storage-port";
 import { ProjectIssueComposerSubmitBody } from "@/core/domain/project/project-contract.types";
 
+import { ContributionReactQueryAdapter } from "../../contribution";
```

ESLint

[error] 'ContributionReactQueryAdapter' is defined but never used.

(unused-imports/no-unused-imports)